thirdeye-server/src/main/java/ai/startree/thirdeye/resources/AlertResource.java

```diff
@@ -257,4 +266,29 @@ private long safeEndTime(final @Nullable Long endTime) {
     }
     return endTime;
   }
+
+  @GET
+  @Timed
+  @Path("{id}/stats")
+  @Produces(MediaType.APPLICATION_JSON)
+  public Response getAnalytics(
+      @ApiParam(hidden = true) @Auth ThirdEyePrincipal principal,
+      @PathParam("id") final Long id,
+      @QueryParam("enumerationItem.id") final String enumerationId,
+      @QueryParam("startTime") final String startTime,
+      @QueryParam("endTime") final String endTime
+  ) {
+    ensureExists(id);
+    final MultivaluedMap<String, String> queryParameters = new MultivaluedHashMap<>();
+    // required filter
+    queryParameters.put("alert.id", List.of(String.valueOf(id)));
+
+    // optional filters
+    optional(enumerationId).map(enumId -> queryParameters.put("enumerationItem.id", List.of(enumId)));
+    optional(startTime).map(start -> queryParameters.put("startTime", List.of(start)));
+    optional(endTime).map(end -> queryParameters.put("endTime", List.of(end)));
+
+    final DaoFilter filter = new DaoFilterBuilder(AnomalyResource.API_TO_INDEX_FILTER_MAP).buildFilter(queryParameters);
```

Comment:
Why not provide the predicate directly?

This is pretty roundabout. 
```
QueryParam > Map > DaoFilterBuilder > DaoFilter.
```

Let's simplify QueryParam > DaoFilter.

Reply:
Query params with operators such as `startTime=[gte]166197060000` needs to be parsed to get something like Predicate.GE("startTime", 166197060000). Currently this operator map is private to DaoFilterBuilder and the only public method DaoFilterBuilder exposes is the one that takes query params map to build DaoFilter. Let me know if I am missing something here.
Else we may need to refactor the DaoFilterBuilder to skip the roundabout.

Reply:
Removed the complexity of having operators in query params. It makes sense to just accept the start and end time without operators. Was thinking about making it consistent to anomaly filtering api call hence had the params in query initially.